src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
+const DAMAGE_AMOUNT = 50;
+const HEALTH_AMOUNT = 100;
```

Comment:
It's `DEFAULT_HEALTH` I think, as if you path another to constructor the health amount will be different